_data-prepper/pipelines/pipelines.md

```diff
@@ -84,269 +88,8 @@ conditional-routing-sample-pipeline:
-    - grok:
-        match:
-          request: ["(%{NOTSPACE:http_method})? (%{NOTSPACE:http_uri})? (%{NOTSPACE:http_version})?"]
-    - grok:
-        match:
-          http_uri: ["(%{WORD:protocol})?(://)?(%{IPORHOST:domain})?(:)?(%{INT:http_port})?(%{GREEDYDATA:request_uri})?"]
-    - date:
-        from_time_received: true
-        destination: "@timestamp"
-
-
-  sink:
-    - opensearch:
-        hosts: [ "https://localhost:9200" ]
-        username: "admin"
-        password: "admin"
-        index: alb_logs
-```
-
-## Migrating from Logstash
-
-Data Prepper supports Logstash configuration files for a limited set of plugins. Simply use the logstash config to run Data Prepper.
-
-```bash
-docker run --name data-prepper \
-    -v /full/path/to/logstash.conf:/usr/share/data-prepper/pipelines/pipelines.conf \
-    opensearchproject/opensearch-data-prepper:latest
-```
-
-This feature is limited by feature parity of Data Prepper. As of Data Prepper 1.2 release, the following plugins from the Logstash configuration are supported:
-
-- HTTP Input plugin
-- Grok Filter plugin
-- Elasticsearch Output plugin
-- Amazon Elasticsearch Output plugin
-
-## Configure the Data Prepper server
-
-Data Prepper itself provides administrative HTTP endpoints such as `/list` to list pipelines and `/metrics/prometheus` to provide Prometheus-compatible metrics data. The port that has these endpoints has a TLS configuration and is specified by a separate YAML file. By default, these endpoints are secured by Data Prepper docker images. We strongly recommend providing your own configuration file for securing production environments. Here is an example `data-prepper-config.yaml`:
-
-```yml
-ssl: true
-keyStoreFilePath: "/usr/share/data-prepper/keystore.jks"
-keyStorePassword: "password"
-privateKeyPassword: "other_password"
-serverPort: 1234
-```
-
-To configure the Data Prepper server, run Data Prepper with the additional yaml file.
-
-```bash
-docker run --name data-prepper \
-    -v /full/path/to/my-pipelines.yaml:/usr/share/data-prepper/pipelines/my-pipelines.yaml \
-    -v /full/path/to/data-prepper-config.yaml:/usr/share/data-prepper/data-prepper-config.yaml \
-    opensearchproject/data-prepper:latest
-```
-
-## Configure peer forwarder
```

Comment:
Deleted this example because it's redundant to https://opensearch.org/docs/latest/data-prepper/managing-data-prepper/peer-forwarder/